docs/lib/project-setup/fragments/flutter/null-safety/null-safety.md

```diff
@@ -0,0 +1,20 @@
+**Amplify Flutter and Null Safety**
+
+
+The Amplify Flutter library supports [Dart null safety](https://dart.dev/null-safety) beginning with version 0.2.0.  This means that:
+
+
+
+* Applications utilizing [sound null safety](https://dart.dev/null-safety) must use Amplify Flutter version 0.2.0 or greater.
```

Comment:
```suggestion
* Applications utilizing [sound null safety](https://dart.dev/null-safety) should use Amplify Flutter version 0.2.0 or greater.
```

Technically someone with a null safe app can use any version if they run it with `--no-sound-null-safety`. We actually have that in our docs for people that created apps with flutter 2.2+. I would just change this to "should" since it isn't technically a requirement.